quill-core/src/main/scala/io/getquill/quotation/Parsing.scala

```diff
@@ -350,8 +350,10 @@ trait Parsing extends SchemaConfigParsing {
       Delete(astParser(query))
   }
 
+  private val arrow = pq"→|->"
```

Comment:
[pattern interpolator](http://docs.scala-lang.org/overviews/quasiquotes/syntax-summary.html#patterns) the cleanest way I found to support both arrow without code duplication :)